### PR TITLE
fix: add --parsable to sbatch call for a more robust output parsing

### PR DESCRIPTION
**Pull request by nigiord**
Following discussion in https://github.com/snakemake/snakemake-executor-plugin-slurm/issues/121#issuecomment-2270905243 , this implementation seems more robust than match a word with only numbers (which could break in future versions of slurm, or if a cluster name is made of only numbers).

### CODE REVIEW

**Review by cmeesters — APPROVED** [review]

**Comment by cmeesters** [comment]
Thank you for your contribution! I will merge this (after fixing the linter issue) - but a new release has to wait until some more contribs are merged. This contrib does not change behaviour, after all. It may wait a day or two.

**Comment by nigiord** [comment]
Sorry for the linting issue, I fixed it. Of course the current implementation works fine so it can wait for the next release :) . Thank you for time!

**Comment by cmeesters** [comment]
no need to apologize - thank you for your contribution once more!

(now, after the holidays, I am still pretty busy, but will work on improving the code once more)

**Pull request merged**
